packages/www/docs/guides/upload-a-video/index.mdx

```diff
@@ -0,0 +1,682 @@
+          "pixelFormat": "yuv420p"
+        },
+        {
+          "type": "audio",
+          "codec": "aac",
+          "bitrate": 253375,
+          "channels": 2,
+          "duration": 12.031667,
+          "sampleRate": 48000
+        }
+      ],
+      "duration": 12.031667
+    },
+    "playbackId": "8728n2iaziowj67d",
+    "playbackUrl": "https://livepeercdn.com/recordings/e605988e-e8d3-4e14-b5b0-10b01ff94a8b/index.m3u8",
+    "downloadUrl": "https://livepeercdn.com/asset/8728n2iaziowj67d/video"
+  }
+]
+```
+
+### **Export an asset to IPFS**
+
+This method allows for exporting an asset imported or uploaded from Livepeer to
+IPFS
+
+Provide the `$ASSET_ID` of the asset, this can be found using the List Asset
+method
+
+Js + axios
+
+```jsx
+const axios = require("axios");
+
+axios
+  .post("https://livepeer.com/api/asset/$ASSET_ID/export", {
+    headers: {
+      Authorization: "Bearer {api_key}",
+      "Content-Type": "application/json",
+    },
+    data: JSON.stringify({
+      ipfs: {},
+    }),
+  })
+  .then((res) => {
+    console.log(JSON.stringify(res.data));
+  })
+  .catch((error) => console.log(error));
+```
+
+Response
+
+The response shows that is it pending
+
+```json
+{
+  "task": {
+    "id": "8b0cd796-8be1-412b-b1dc-8251a35ad2e7",
+    "createdAt": 1652903903963,
+    "type": "export",
+    "inputAssetId": "87289307-b60f-432b-984a-8ede174fa152",
+    "userId": "eada599f-3d58-499b-ba24-c7f3faf988de",
+    "params": {
+      "export": {
+        "ipfs": {}
+      }
+    },
+    "status": {
+      "phase": "pending",
+      "updatedAt": 1652903903963
+    }
+  }
+}
+```
+
+Check back with `https://livepeer.com/api/task/$TASK_ID` to get the status
+update
+
+```json
+{
+  "id": "8b0cd796-8be1-412b-b1dc-8251a35ad2e7",
+  "type": "export",
+  "output": {
+    "export": {
+      "ipfs": {
+        "videoFileCid": "QmaMvRNEwvCHuwEZjX9NzFTJtBUrbk7j5D7ngP8xATHwXK",
+        "nftMetadataCid": "QmaY8HqYV3kVxwsCQahMYisQiZnjZg2bRVjb6XspTmYMFi",
+        "videoFileUrl": "ipfs://QmaMvRNEwvCHuwEZjX9NzFTJtBUrbk7j5D7ngP8xATHwXK",
+        "videoFileGatewayUrl": "https://ipfs.livepeer.com/ipfs/QmaMvRNEwvCHuwEZjX9NzFTJtBUrbk7j5D7ngP8xATHwXK",
+        "nftMetadataUrl": "ipfs://QmaY8HqYV3kVxwsCQahMYisQiZnjZg2bRVjb6XspTmYMFi",
+        "nftMetadataGatewayUrl": "https://ipfs.livepeer.com/ipfs/QmaY8HqYV3kVxwsCQahMYisQiZnjZg2bRVjb6XspTmYMFi"
+      }
+    }
+  },
+  "params": {
+    "export": {
+      "ipfs": {}
+    }
+  },
+  "status": {
+    "phase": "completed",
+    "updatedAt": 1652903908984
+  },
+  "userId": "eada599f-3d58-499b-ba24-c7f3faf988de",
+  "createdAt": 1652903903963,
+  "inputAssetId": "87289307-b60f-432b-984a-8ede174fa152"
+}
+```
+
+### **List Tasks**
+
+When listing tasks, the response will provide information on imported or
+exported information on the asset
+
+Use the list tasks method to get CID and metadata
+
+To Get the CID and metadata URL for the asset that was just exported,  you can
```

Comment:
Change to 

To get the CID and metadata URL for the new, just created asset, retrieve the task until it is completed. To do this, check the **status** field in the response. This finished task will have an output with all the information about that asst, and the CID information is located under the `output.export.ipfs` path.